libqtile/widget/widgetbox.py

```diff
@@ -27,8 +28,12 @@
 BoxedWidget = namedtuple("BoxedWidget", ["widget", "draw"])
 
 
-def _no_draw(*args, **kwargs):
-    pass
+def _no_draw(self, *args, **kwargs):
```

Comment:
This looks a lot like a method now! Perhaps we should move it to the base widget class, then no need for the `partial` stuff?

Reply:
Agreed. Have updated.